services/QuillLMS/Gemfile

```diff
@@ -144,6 +144,7 @@ gem 'sentry-ruby'
 gem 'sentry-rails'
 gem 'sentry-sidekiq'
 gem 'rack-heartbeat'
+gem "matrix", '~> 0.4.2'
```

Comment:
If this is only for heroku deploy, let's move this to the :production, :staging group